src/fields.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+function add_fields(data, datatype) {
+				var div = document.getElementById(data);
```

Comment:
Use a variable name which are not keywords in HTML, e.g., 'div' and 'input', for readability purpose.